src/components/LogIn/LogIn.css

```diff
@@ -1,9 +1,59 @@
-body{
+.loginBody{
     background-color: black;
     overflow: hidden;
     margin: 0;
 }
 
-.fontFamily{
+#loginBox {
+    opacity: 0;
+    transition: opacity 1.5s;
+
```

Comment:
We might also want to think about rounding the buttons and the text fields to match how the box is also rounded. That is a design change so I think that it should be mentioned to the group before anything is set in stone. I personally found that the attribute set to border-radius: 0.5rem; seemed to work well with the page. The only thing with that is the placeholder text might need to be spaced more towards the center. 
![image](https://user-images.githubusercontent.com/10992950/92340754-976e5e80-f089-11ea-95af-7ff658735935.png)

Reply:
7c3a5bc